runtime/src/accounts_db.rs

```diff
@@ -9923,10 +9968,11 @@ pub mod tests {
         let accounts = AccountsDb::new_single_for_tests();
 
         let mut candidates: ShrinkCandidates = HashMap::new();
-        let output_candidates =
+        let (output_candidates, next_candidates) =
             accounts.select_candidates_by_total_usage(&candidates, DEFAULT_ACCOUNTS_SHRINK_RATIO);
 
         assert_eq!(0, output_candidates.len());
+        assert_eq!(0, next_candidates.len());
```

Comment:
I think as this list of test cases grows, it's good to break these tests into separate tests with the prefix `test_select_candidates_{description of test}`

Reply:
Addressed